src/data/csv/assembly/2024-H2.csv

```diff
@@ -0,0 +1,16 @@
+日期,講員,主題,視頻/錄音.video,視頻/錄音
```

Comment:
These should be added to the upcoming.csv. 2024-H2.csv should be created when the first video is posted. And it should be linked in assembly.mdx

Reply:
This week 8/21 assembly video was updated on upcoming.csv

Reply:
This file should be linked to the Assembly.mdx. And it should contain the posted items. Other scheduled items should go upcoming.csv. The spreadsheet was also edited in a wrong way. I fixed it and set the permission not allowing the title row being changed.